{{cookiecutter.project_slug}}/frontend/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```

Comment:
Can we not include the project title here?

Reply:
Maybe `{{cookiecutter.project_name}}` or `{{cookiecutter.project_name}} - {{cookiecutter.description}}`